### DISCUSSION TITLE
How to set default style for all levels - node, edge, connectors

### DISCUSSION THREAD

**Discussion by LivingInternet** [freeplane/freeplane#1942] [Help]
I am using FreePlane 1.11.14. I followed the direction to set standard styles [here](https://sourceforge.net/p/freeplane/discussion/758437/thread/c220ae649f/).  For the default node, I set the style to rectangular and light grey background.  For default edges, I set the style to horizontal and light grey. And then saved the styles.  Please see attached test file.  [test3.zip](https://github.com/user-attachments/files/16417412/test3.zip)

Now when I create a new node, they have the right style. But the connectors are always a new colour.  So I guess setting the edge style was not sufficient

Is it possible to set the default style for everything, node edges and connector lines, for example to always be light grey?

**Comment by macmarrum** [comment]
You have **Automatic edge color** set to **on branch creation**. 
Look in `View->Controls->Tool panel`, in the top section.
You'll need to reset edge color individually for each already created node.

**Comment by LivingInternet** [comment]
That solves it.  Thank you very much.  

Fwiw, FreePlane provides the only real open source tool that can create something very close to organization charts, i.e. vertical hierarchies.  I have found that by using the drop down vertical format, node shape "rectangle", edge style "horizontal", making the nodes the same size by setting the minimum width all to the same size (I usually use 2.2 cm), and then putting in a carriage return and blank line for one line entries so they are all two lines high, you get something very very much like a conventional org chart style.  

PowerPoint has an amazingly useful org chart / hierarchy tool, but LibreOffice Impress does not.  The world has been looking for an open source org chart tool for a very long time.  In project management, we use them a lot, not for org charts, but for what are called Work Breakdown Structures.  I've been looking for an open source solution for 20 years.  This is close enough.  Thanks to all.

![image](https://github.com/user-attachments/assets/f055e3eb-2b8f-43f0-95df-0b2b498f0ff4)

I provided a full description of all the settings to make using FreePlane to create organization charts or vertical hierarchies easy in [this comment](https://www.reddit.com/r/opensource/comments/1ekqkzq/how_to_make_organization_charts_and_hierarchies/).
